examples/docker-compose/package.json

```
@@ -6,6 +6,8 @@
     "e2e": "vitest run"
   },
   "devDependencies": {
-    "cross-fetch": "^3.1.5"
+    "@nhost/nhost-js": "workspace:*",
```

```suggestion
    "@nhost/nhost-js": "*",
```

I think we want to use `*` here because users might want to copy the project outside of the repository where `workspace:*` is not available. I don't think this is a frequent use case, but I remember we talked about this in the past and decided to go with `*`.